src/js/index.js

```diff
@@ -43,20 +43,20 @@ export const hpe = deepFreeze({
       },
       icon: 'text',
       text: {
-        dark: '#FFFFFF',
-        light: '#333333',
+        dark: '#DDDDDD',
```

Comment:
This change would break the accessibility contrast standard for primary Button.
https://webaim.org/resources/contrastchecker/?fcolor=DDDDDD&bcolor=01A982